shared/src/store/action-reducers/vehicle.ts

```diff
@@ -294,6 +300,18 @@ export namespace VehicleActionReducers {
                         // If a personnel is in transfer, this doesn't change that
                         draftState.personnel[personnelId].position = undefined;
                     });
+                    if (
```

Comment:
I don't like this solution for `dev`, but for an experimental branch, it should be fine.